src/view.js

```diff
@@ -1,6 +1,8 @@
 import * as Tone from 'tone';
 import { toSentenceCase } from './utils';
 
+const levels = ['Easy', 'Intermediate', 'hard'];
```

Comment:
```suggestion
const levels = ['Easy', 'Intermediate', 'Hard'];
```
We uppercase the first letter in our buttons so we won't see this in the UI, but let's keep this consistent 😅